src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/client/caseupdate/CaseUpdateClientResponseValidationTest.java

```diff
@@ -0,0 +1,120 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.client.caseupdate;
+
+import com.google.common.collect.ImmutableMap;
+import org.assertj.core.groups.Tuple;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.BDDMockito;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.web.client.RestTemplate;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.client.caseupdate.model.response.CaseUpdateDetails;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.client.caseupdate.model.response.SuccessfulUpdateResponse;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.client.model.request.ExceptionRecord;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+
+import javax.validation.ConstraintViolationException;
+import javax.validation.Validation;
+import javax.validation.Validator;
+
+import static java.util.Collections.emptyList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatCode;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+import static org.assertj.core.api.Assertions.tuple;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.mock;
+
+@ExtendWith(MockitoExtension.class)
+class CaseUpdateClientResponseValidationTest {
+
+    @Mock RestTemplate restTemplate;
+    Validator validator = Validation.buildDefaultValidatorFactory().getValidator();
```

Comment:
Suggestion: we only reference this validator once, in `setUp`. Could as well define it there.